src/mem/epoch/mod.rs

```diff
@@ -406,7 +408,7 @@ thread_local!(static LOCAL_EPOCH: LocalEpoch = LocalEpoch::new() );
 
 #[must_use]
 pub struct Guard {
-    _dummy: ()
+    _marker: marker::PhantomData<Rc<()>>, // !Send and !Sync
```

Comment:
Why `Rc` rather than, say, `*mut ()`? To avoid the extra `use`, if nothing else.